src/nodes/typeGuards/compound.ts

```diff
@@ -9,15 +9,32 @@ import {
 	isNamespaceBody,
 } from "./union.js";
 
+/**
+ * An `AssertionExpression` that is declared as const.
+ *
+ * @category Node Types
+ */
 export type ConstAssertionExpression = ts.AssertionExpression & {
 	type: ts.TypeReferenceNode;
 	typeName: ConstAssertionIdentifier;
 };
 
+/**
+ * An `Identifier` with an `escapedText` value of `"const"`.
+ *
+ * @category Node Types
+ */
 export type ConstAssertionIdentifier = ts.Identifier & {
 	escapedText: ts.__String & "const";
 };
 
+/**
+ * Test if a node is a {@link ConstAssertionExpression}.
+ *
+ * @category Nodes - Type Guards
+ * @param node - The node in question.
```

Comment:
[Nitpick] I don't think this description adds anything.

```suggestion
```

_(here and elsewhere)_